Package.swift

```diff
@@ -35,7 +35,7 @@ let products: [Product] = [
 let dependencies: [Package.Dependency] = [
   .package(
     url: "https://github.com/grpc/grpc-swift.git",
-    from: "2.0.0"
+    branch: "main"
```

Comment:
This is temporary so that CI runs, we'll need bump to a version containing https://github.com/grpc/grpc-swift/pull/2201 once it's released.